src/git.js

```diff
@@ -1,8 +1,12 @@
 import util from "./utils";
+import path from "path";
+import semver from "semver";
+
+const DEFAULT_PRERELEASE_TAG_LIST_LIMIT = 10;
 
 const git = {
-	runCommand( { args, showOutput = true, logMessage, failHelpKey = "gitCommandFailed", exitOnFail = false } ) {
-		const command = `git ${ args }`;
+	runCommand( { args, showOutput = true, logMessage, failHelpKey = "gitCommandFailed", exitOnFail = false, showError = true, fullCommand = false } ) {
```

Comment:
In a future PR we may consider pulling out `runCommand` into the `utils` and then having this `runCommand` be a thin wrapper around it that prepends `"git  "` to the command.